app/main.py

```diff
@@ -1,5 +1,61 @@
-class Cargo:
-    def __init__(self, weight: int) -> None:
-        self.weight = weight
+from typing import List, Optional
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: float,
+                 coords: Optional[List[int]] = None) -> None:
+        self.name: str = name
+        self.weight: float = weight
+        self.coords: List[int] = coords if coords is not None else [0, 0]
```

Comment:
You can use operator `or` to handle `None` value. It makes you code simple.
```suggestion
        self.coords: List[int] = coords or [0, 0]
```